### PR TITLE
Material randomization and moved scene.add_from_file to Mesh class

### LINKED ISSUE #46: Randomize renderer color/materials
```
new_mat = bpy.data.materials.new('Chrome')
new_mat.use_nodes = True
tree = mat.node_tree
[...]
o = bpy.context.object
o.active_material = new_mat
```

Control color as:
```
bpy.data.materials["Metal.002"].node_tree.nodes["Principled BSDF"].inputs[0].default_value = (0.749198, 0.297648, 0.249657, 1)
```

To duplicate materials:
https://blender.stackexchange.com/questions/82768/copy-material-with-python

### PR DESCRIPTION
**Pull request by taiya**

### CODE REVIEW

File: kubric/viewer/blender.py [hunk 1]
```diff
@@ -335,9 +363,30 @@ def __init__(self, geometry: Geometry, material: Material):
       self._blender_object.data.from_pydata(vertices, [], faces)
 
     # --- Adds the material to the object
+    # TODO: why not just use active_material as in set_material method?
```
Comment (Qwlouse) [inline kubric/viewer/blender.py:366]: Blender allows multiple materials per object, for example to assign different materials to different parts of the mesh. 
If I understand correctly, then `materials.append` adds a new slot and fills it with the material. If the object did not have a material before, then this is also assigning the material to the entire mesh. However subsequent `append`s will only add material slots and not assign them (meaning that there is no visible change).
Setting `active_material` changes the active slot to a particular material. If the active slot is assigned to the mesh, then this will change the visible material of the object. 

So in short: Both versions will work if only called once on an object with no material. Setting `active_material` will also work if the object has a single material, or if the right slot is active. A fully general solution would either have to clear the slots first, or explicitly assign the material to the entire mesh.

File: worker.py [hunk 2]
```diff
@@ -140,34 +148,40 @@
...
-  o.position = obj.position
-  o.quaternion = translate_quat(obj.rotation)
-
-  if FLAGS.randomize_color:
-    logging.warning("TODO: color randomization")
-    pass
-
+  # --- Load the mesh into the scene
+  mesh = THREE.Mesh.from_file(str(obj.vis_filename), name=obj.uid)
+  mesh.position = obj.position # TODO: why set? see keyframing below
```
Comment (Qwlouse) [inline worker.py:167]: Both in our code and in Blender there is a distinction between the "default" position of an object and the keyframed position. It is a bit strange, but seems to be pretty standard. 
For rendering the animation this is irrelevant though.

File: worker.py [hunk 3]
```diff
@@ -140,34 +148,40 @@
...
+  mesh.quaternion = obj.rotation # TODO: why set? see keyframing below
```
Comment (Qwlouse) [inline worker.py:168]:
```suggestion
  mesh.quaternion = translate_quat(obj.rotation) # TODO: why set? see keyframing below
```

File: kubric/viewer/blender.py [hunk 4]
```diff
@@ -298,6 +299,33 @@ def blender_apply(self, blender_object):
...
+    LW = tree.nodes.new('ShaderNodeLayerWeight')
+    LW.inputs[0].default_value = 0.7
+    CR = tree.nodes.new('ShaderNodeValToRGB')
+    CR.color_ramp.elements[0].position = 0.9
+    CR.color_ramp.elements[0].color = color_rgba
+    CR.color_ramp.elements[1].position = 1
+    CR.color_ramp.elements[1].color = (0,0,0,1)
+    GLO = tree.nodes.new('ShaderNodeBsdfGlossy')
+    GLO.inputs[1].default_value = roughness
+    GLO.location.x -= 200  # TODO: ???
```
Comment (Qwlouse) [inline kubric/viewer/blender.py:321]: Affects only the visual location of the node if using the Blender GUI. Can be nice for manually debugging a `.blend` file, but doesn't affect the rendering in any way.

File: requirements.txt [hunk 5]
```diff
@@ -10,3 +10,4 @@ OpenEXR
 pytest
 mathutils
 dataclasses
+colorsys
```
Comment (Qwlouse) [inline requirements.txt:13]: The `colorsys` package is part of the standard library and cannot be installed via pip. (This is why the tests fail)

```suggestion

```

File: kubric/viewer/interface.py [hunk 6]
```diff
@@ -31,6 +31,8 @@ def hex_to_rgba(hexint: int, alpha=1.0):
   r = (hexint >> 16) & 255
   return [r / 255.0, g / 255.0, b / 255.0, alpha]
 
+def hsv_to_rgba(color, alpha=1.0):
```
Comment (Qwlouse) [inline kubric/viewer/interface.py:34]: This isn't even needed anymore is it?

File: worker.py [hunk 7]
```diff
@@ -115,6 +119,10 @@
 renderer.set_up_background(bg_color=(0., 0., 0., 0.))
 renderer.set_up_exr_output(path=FLAGS.outpath)
 
+# --- Environment settings for CLEVR
+room = THREE.Mesh.from_file(str(floor.vis_filename))
+scene.add(room)
```
Comment (Qwlouse) [inline worker.py:124]: You are adding the floor already below, so this is duplicate
```suggestion

```

**Review by Qwlouse — CHANGES_REQUESTED** [review]

**Review by Qwlouse — APPROVED** [review]
